src/components/rangeUsePlanPage/versionsList/VersionsDropDown.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useRef } from 'react'
```

Comment:
Can we move these two new components to `src/components/selectRangeUsePlanPage`, since that's the only place they're used?

Reply:
@okanji Just a heads up, you missed this change.